programs/drift/src/validation/user.rs

```diff
@@ -52,13 +52,10 @@ pub fn validate_user_deletion(user: &User, user_stats: &UserStats) -> DriftResul
     Ok(())
 }
 
-pub fn validate_user_is_idle(user: &User, slot: u64) -> DriftResult {
+pub fn validate_user_is_idle(user: &User, slot: u64, accelerated: bool) -> DriftResult {
     let slots_since_last_active = slot.saturating_sub(user.last_active_slot);
 
-    #[cfg(feature = "mainnet-beta")]
-    let slots_before_idle = 1512000_u64;
-    #[cfg(not(feature = "mainnet-beta"))]
-    let slots_before_idle = 0_u64;
+    let slots_before_idle = if accelerated { 3600_u64 } else { 1512000_u64 };
```

Comment:
1. since its slots (1 hour is ~2x)
2. can we leave a comment on the estimated intended time for each number?